cmd/update.go

```diff
@@ -14,22 +14,22 @@ import (
 var updateCmd = &cobra.Command{
 	Use:  "update",
 	Args: cobra.MaximumNArgs(1),
-	Example: `  # Change the source of a currently active component to local (use the current directory as a source)
+	Example: `  # Change the source code of a currently active component to local (use the current directory as a source)
```

Comment:
This sound a bit weird to me.
"Change the source code " sounds to me like it is changing the actual source code (lines of code).

I can't comment on the correctness of the previous formulation.
English is not really my forte :-) 

How about "Change the source code path ..." ?

Reply:
@kadel that's a better option :+1: I changed it to that!